attributes/default.rb

```diff
@@ -112,9 +112,9 @@
 default['cfncluster']['torque']['url'] = 'https://github.com/adaptivecomputing/torque/archive/6.1.2.tar.gz'
 # Slurm software
 default['cfncluster']['slurm_plugin_dir'] = '/etc/parallelcluster/slurm_plugin'
-default['cfncluster']['slurm']['version'] = '20.11.5'
-default['cfncluster']['slurm']['url'] = 'https://download.schedmd.com/slurm/slurm-20.11.5.tar.bz2'
-default['cfncluster']['slurm']['sha1'] = '201a28afe6f02a717fb348542878900cad4ccf13'
+default['cfncluster']['slurm']['version'] = '20.11.7'
```

Comment:
Please double check this. We use this version to interpolate the slurm tarball name and extracted name.
The downloaded tarball from github will have different name. And the extracted folder will be named slightly different as well.
I believe the version should be `20-11-7-1`, default downloaded tarball filename is `slurm-20-11-7-1.tar.gz`, and default extracted folder name is `slurm-slurm-20-11-7-1`.

Reply:
Fixed thanks

Reply:
Fixed thanks. Patch still under testing.